sdk/textanalytics/azure-ai-textanalytics/samples/README.md

```diff
@@ -1,31 +1,34 @@
 ---
-topic: sample
+page_type: sample
 languages:
   - python
 products:
   - azure
   - azure-ai-textanalytics
+urlFragment: cognitiveservices-textanalytics-samples
```

Comment:
It feels weird that we never mentioned Cognitive Services anywhere in our docs and suddenly it is here,.
True that it is the umbrella service, but not sure if we should use it here.
Do you know what this `urlFragment` does? I didn't find it in our .NET repo for any library

Reply:
The guidance I see from [here](https://microsoft.sharepoint.com/teams/AzureDeveloperExperience/_layouts/15/Doc.aspx?sourcedoc={33bd5249-4696-4382-b9f5-c5c1436a2b36}&action=edit&wd=target%28Vanadium%20Planning%20and%20Status.one%7Cf621d863-c3ec-4e47-9e60-08ed695aa5ab%2FSamples%20Guidance%7C2daef4da-3fd3-4fce-834d-d7f98ef72dc4%2F%29) is that it's {service}-{package}-samples. I pinged someone about this url fragment yesterday, I'll reping them and get back to you. thanks!

Reply:
thanks for raising the concern, after talking to the docs team, I'm going to make it just textanalytics-samples

Reply:
Thanks for following up on that!